Assets/Scripts/Frame.cs

```diff
@@ -1,189 +0,0 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.Serialization.Formatters;
-using UnityEngine;
-using UnityEngine.Video;
-
-public class Frame
-{
-    public static List<Frame> FrameList = new List<Frame>();
-    public static List<int> TotalRolls = new List<int>();
-    public static List<int?> FrameScores = new List<int?>();
```

Comment:
Ah ok so a general rule of thumb is - when you see static - someone is doing something a bit naughty. lol.

Static causes problems because the state isn't cleaned up in the ways we normally think of. When the object is removed, or the game transitions between states all of these variables will keep their values. Sometimes we want that. Sometimes we forget.

It also promotes you "just using it" from all over the code base. Slowly you'll notice ```Frame.TotalRoles``` and ```Frame.FrameList``` sneeking in all over the place, and then when you want to change which frame a particular class is operating on, it becomes tricky.